sycl/test-e2e/syclcompat/math/math_vectorized.cpp

```diff
@@ -101,6 +146,27 @@ int main() {
                                                         0xFFF8FFFD);
   test_vectorized_unary<syclcompat::abs, uint32_t>(0xFFFBFFFD, 0x00050003);
   test_vectorized_sum_abs_diff<uint32_t>(0x00010002, 0x00040002, 0x00000003);
+  test_vectorized_ternary<std::plus<>, syclcompat::maximum, uint32_t>(
+      0x00010002, 0x00040002, 0x00080004, 0x00030000);
+  test_vectorized_ternary<std::plus<>, syclcompat::maximum, uint32_t>(
+      0x00010002, 0x00040002, 0x00080004, 0x00030000, true);
+  test_vectorized_ternary<std::plus<>, syclcompat::minimum, uint32_t>(
+      0x00010002, 0x00040002, 0x00080004, 0x00030000);
+  test_vectorized_ternary<std::plus<>, syclcompat::minimum, uint32_t>(
+      0x00010002, 0x00040002, 0x00080004, 0x00030000, true);
+  test_vectorized_ternary<syclcompat::maximum, syclcompat::maximum, uint32_t>(
+      0x00010002, 0x00040002, 0x00080004, 0x00030000);
+  test_vectorized_ternary<syclcompat::maximum, syclcompat::maximum, uint32_t>(
+      0x00010002, 0x00040002, 0x00080004, 0x00030000, true);
+  test_vectorized_ternary<syclcompat::minimum, syclcompat::minimum, uint32_t>(
+      0x00010002, 0x00040002, 0x00080004, 0x00030000);
+  test_vectorized_ternary<syclcompat::minimum, syclcompat::minimum, uint32_t>(
+      0x00010002, 0x00040002, 0x00080004, 0x00030000, true);
```

Comment:
It feels strange that all the new tests return the same values with the same operators despite the operations being different. Could you please double check if that is the case?